toolkits/nature/packages/nature-footer/package.json

```diff
@@ -1,9 +1,9 @@
 {
     "name": "@springernature/nature-footer",
-    "version": "1.0.0",
+    "version": "2.0.0",
     "license": "MIT",
     "description": "Nature branded footer",
     "keywords": [],
     "author": "Springer Nature",
-    "brandContext": "^9.0.1"
+    "brandContext": "^17.0.0"
```

Comment:
should this be the latest `18.0.1`?

Reply:
I found an issue with Travis yesterday when I set everything to `^18.0.0` so dropped it back to `^17.0.0` to make Travis happy.

@benjclark suggested a doing quick 'post merge' PR to move this to `^18.0.0` for the components, which I didn't get to because we found the font size issues with Oscar.

I _think_ the `^` means it should look for `v18.0.0` so perhaps we merge this in now and I go through and make the changes to `18.0.1` on the components after the brand context os updated to npm?

Reply:
Oh of course yes, because the v18 won't be published at the point that travis checks if it exists for the other components. Would these changes not break the font sizing if used with 17?

Reply:
hmmm, I guess they might do. 

As `18.0.0` is now live on NPM I could change any instances of `^17.0.0` I have made to `^18.0.0` and that'll clear it up?

Reply:
Yep let's go with that